crate/multiworld/src/localisation.rs

```diff
@@ -0,0 +1,62 @@
+use {
+    serde::{
+        Deserialize,
+        Serialize,
+    }, std::fmt
+};
+
+#[derive(Clone, Copy, Debug, Default, Deserialize, Serialize, PartialEq, clap::ValueEnum)]
+#[clap(rename_all = "lower")]
+pub enum Locale {
+    #[default]
+    EN,
+    FR,
+}
+
+impl Locale {
+    pub const ALL: [Locale; 2] = [
+        Locale::EN,
+        Locale::FR,
+    ];
```

Comment:
Instead of hardcoding this list, I recommend deriving [`enum-iterator::Sequence`](https://docs.rs/enum-iterator/latest/enum_iterator/trait.Sequence.html) on `Locale`. Usages of `Locale::ALL` can then be replaced with `enum_iterator::all()` (or `all::<Locale>()` if Rust can't infer the type).